src/GlaceView.mc

```diff
@@ -0,0 +1,104 @@
+using Toybox.System;
+using Toybox.Timer;
+using Toybox.WatchUi;
+
+using TextInput;
+
+(:glance)
+class WidgetGlanceView extends WatchUi.GlanceView {
```

Comment:
Typo in file name GlaceView.mc -> GlanceView.mc (or WidgetGlanceView.mc == class name)